.gitignore

```diff
@@ -11,3 +11,4 @@ out
 sqldelight-gradle/plugin/src/test/integration/gradle
 sqldelight-gradle/android-plugin/src/test/integration/gradle
 sqldelight-gradle/android-plugin/src/test/schema-file/src/main/sqldelight/databases/1.db
+.DS_Store
```

Comment:
You can put this in your global ignores. It shouldn't need to be copied to each project.

Reply:
TIL! Reverted